docs/source/conf.py

```diff
@@ -18,8 +18,11 @@
 sys.path.insert(0, os.path.abspath('../..'))
 
 project = '5yncr'
-copyright = '2018, Brett Johnson, David Lance, Jack LaRue, Mathew Bentley, Alexander Tryjankowski'
-author = 'Brett Johnson, David Lance, Jack LaRue, Mathew Bentley, Alexander Tryjankowski'
+copyright = '''2018, Alexander Tryjankowski Brett Johnson,
```

Comment:
comma after Tryjankowski, my name is still misspelled, and alphabetical by last name.